src/helpers.rs

```diff
@@ -186,33 +197,33 @@ pub fn create_decrypt_config(
 
     let [_, _, mut x509s, _, _, _] = process_recipient_keys(dec_recipients)?;
 
-    if x509s.len() > 0 {
+    if !x509s.is_empty() {
         let x509_from_keys = process_x509_certs(keys.clone())?;
```

Comment:
Minor inconvenience here... There was a bug in the original implementation here that was recently fixed, I think this is the relevant piece of code, https://github.com/containers/ocicrypt/pull/54

Reply:
Thanks, I'll also fix it in this PR.